src/main/resources/add-ons-to-index.json

```diff
@@ -1,5 +1,26 @@
 {
   "toIndex": [
+    {
+      "uid": "org.openmrs.owa.orderentryui",
+      "type": "OWA",
+      "name": "Order Entry UI OWA",
+      "maintainers": [
+        {
+          "name": "Andela apprentices",
+          "url": "https://andela.com/"
+        },
+        {
+          "name": "OpenMRS",
```

Comment:
Are you sure we can should OpenMRS as a maintainer?  What do you feel @dkayiwa @djazayeri ?